templates/kickstart.j2

```diff
@@ -6,15 +6,24 @@ network --noipv6
 # Ansible has no way of accessing a group's variables, so they are accessed
 # via the first host of the group.
 
+{% if hostvars[groups[item][0]]['yum_proxy'] is defined %}
+url --url {{ hostvars[groups[item][0]]['install_repo'] }} --proxy={{ hostvars[groups[item][0]]['yum_proxy'] }}
```

Comment:
I'm a bit rusty but, what is the reason for using `hostvars[groups[item][0]]['install_repo']` instead of `install_repo` ?

Reply:
When I tested with those cleaner variable names, for some reason the precedence of variables in various places wasn't the same as with these hostvars[]. Why that happens I'm not sure. In the PR from Sonera it looks like they found a way.

I tested initially in this PR with the commit before https://github.com/CSCfi/ansible-role-dhcp-kickstart/pull/17/commits/1ac29fbb6065b4f41bec6e07dd5380e900cad591